nginx/tripleawarclub.org

```diff
@@ -29,11 +29,6 @@ server {
     index  index.html index.htm index.php;
 	
     location / {
-        try_files $uri $uri/ =404;
-    }
-
-    location ~ .*\.php$ {
-        fastcgi_pass unix:/var/run/php/php7.0-fpm.sock;
-        include snippets/fastcgi-php.conf;
+        return 301 https://forums.triplea-game.org;
     }
```

Comment:
As already mentioned earlier, we need to make an exception for certbot certificates, so `/.well-known` requests aren't being redirected, otherwise we're going to run into certificate issue pretty soon.
So of you have the time, feel free to propose such a change, otherwise I'm going to propose it on Thursday (or late Wednesday, depending on when I'm back at my PC)

Reply:
Also there's probably a lot of unused settings in this config left, and the server blocks can probably be merged as well, so removing those uneccessary things would probably be a good idea.